src/lib/gossip_net/gossip_net.ml

```diff
@@ -267,7 +271,8 @@ module Make (Message : Message_intf) : S with type msg := Message.msg = struct
           ; peers= Peer.Hash_set.create ()
           ; peers_by_ip= Hashtbl.create (module Unix.Inet_addr)
           ; connections= Hashtbl.create (module Unix.Inet_addr)
-          ; max_concurrent_connections= config.max_concurrent_connections }
+          ; max_concurrent_connections= config.max_concurrent_connections
+          ; initial_peers= config.initial_peers }
```

Comment:
I would move this up to group it with the other peer-related fields, maybe just before `peers`